openwisp_ipam/tests/test_admin.py

```diff
@@ -426,3 +426,12 @@ def assert_response(response):
                 reverse('admin:ipam_export_subnet', args=[subnet.id]), follow=True
             )
             assert_response(response)
+
+    def test_subnet_32(self):
+        subnet = self._create_subnet(subnet='192.168.0.0/32', description='Subnet 32')
+        try:
+            response = self.client.get(reverse('ipam:hosts', args=(subnet.id,)))
+            self.assertIsNone(response.data['next'])
+            self.assertIsNone(response.data['previous'])
+        except ValueError as err:
+            self.fail(f'subnet/32: {err}')
```

Comment:
I don't think we need a try-except block here :eyes: Tests will fail in case of error

Reply:
Right